main.js

```diff
@@ -0,0 +1,48 @@
+document.getElementById("searchButton").addEventListener("click", function () {
+  var locationInput = document.getElementById("locationInput").value;
+  fetchWeatherData(locationInput);
+});
+
+function fetchWeatherData(location) {
+  var apiKey = "235b291d30b715c64c5632322d291bc1";
+  var apiUrl = `http://api.openweathermap.org/data/2.5/forecast?q=${location}&appid=${apiKey}&units=imperial`;
+
+  fetch(apiUrl)
+    .then((response) => response.json())
+    .then((data) => {
+      displayWeatherData(data);
+    });
+}
+
+function displayWeatherData(data) {
+  var weatherIconElement = document.querySelector(".weather-icon");
+  var weatherCardInfo = document.getElementById("weatherCard");
+
+  // Clear previous content
+  weatherIconElement.innerHTML = "";
+
+  // Display current weather
+  var currentWeather = data.list[0];
+  weatherIconElement.innerHTML += `<h3>Current Weather</h3>
+    <p>Temperature: ${currentWeather.main.temp} &deg;F</p>
+    <p>Description: ${currentWeather.weather[0].description}</p>
+    <p>Humidity: ${currentWeather.main.humidity}%</p>`;
+
+  // Display 5-day forecast
+  weatherIconElement.innerHTML += `<h3>5-Day Forecast</h3>`;
+  for (let i = 0; i < 5; i++) {
```

Comment:
let is post es6 syntax - would recommend staying consistent with your variable declarations